src/rt/sections_android.d

```diff
@@ -69,6 +69,8 @@ void initSections() nothrow @nogc
 
     auto pbeg = cast(void*)&_tlsend;
     auto pend = cast(void*)&__bss_end__;
+    // _tlsend is a 32-bit int and may not be 64-bit void*-aligned, so align pbeg.
+    version(D_LP64) pbeg = cast(void*)(cast(size_t)(pbeg + 7) & ~cast(size_t)7);
```

Comment:
`_tlsend` is an integer delimiter added by ldc at the end of TLS data in a compiled object file. The next ELF section after `.tbss` is always 64-bit aligned from all the executables I've seen, so align to the next 64-bit word here. Without aligning, this caused a bug because the GC assumed it was aligned and would iterate over this range in 64-bit steps when scanning and not see any GC pointers. Not a problem for 32-bit because `void*` is 32-bit there, just like `_tlsend`.